code/game/machinery/doors/airlock.dm

```diff
@@ -1083,10 +1088,41 @@ About the new airlock wires panel:
 	use_power(360)	//360 W seems much more appropriate for an actuator moving an industrial door capable of crushing people
 
 	//if the door is unpowered then it doesn't make sense to hear the woosh of a pneumatic actuator
-	if(arePowerSystemsOn())
-		playsound(src, open_sound_powered, 50, 1, volume_channel = VOLUME_CHANNEL_DOORS)
-	else
-		playsound(src, open_sound_unpowered, 75, 1, volume_channel = VOLUME_CHANNEL_DOORS)
+	for(var/P in player_list)
+		var/mob/M = P
+		if(!M || !M.client)
+			continue
+		var/old_sounds = M.client.is_preference_enabled(/datum/client_preference/old_door_sounds)
+		var/department_door_sounds = M.client.is_preference_enabled(/datum/client_preference/department_door_sounds)
+		var/sound
+		var/volume
+		if(old_sounds) // Do we have old sounds enabled? Play these even if we have department door sounds enabled.
+			if(arePowerSystemsOn())
+				sound = legacy_open_powered
+				volume = 50
+			else
+				sound = open_sound_unpowered
+				volume = 75

```

Comment:
Failing the first suggestion, this repeated else should probably instead go in an `if(!arePoweredSystemsOn())` check, the else of which instead contains the if-else that selects among the other sounds.